docs/UserGuide.md

```diff
@@ -60,9 +62,9 @@ Athena is perfect for you if you:
 
 1. Download the latest `Athena.jar` from [here](https://github.com/AY2021S1-CS2103T-W10-4/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for Athena.
+1. Copy the JAR file to the folder you want to use as the _home folder_ for Athena.
 
-1. Double-click the file or run it with `java -jar Athena.jar` to start the app. A window similar to the one below
+1. Double-click the file or run it with `java -jar Athena.jar` on Command Line Interface to start the app. A window similar to the one below
```

Comment:
maybe add that it's your computer's CLI to differentiate it from the app? phrasing now makes it sound like there's one general CLI every computer uses.